ddprof-test/src/test/java/com/datadoghq/profiler/wallclock/ContendedWallclockSamplesTest.java

```diff
@@ -52,9 +52,11 @@ public void after() {
     @ValueSource(strings = {"vm", "vmx", "fp", "dwarf"})
     public void test(@CStack String cstack) {
         // Skip test entirely on unsupported JVMs (don't use assumeFalse which gets retried)
-        if (Platform.isZing() || Platform.isJ9()) {
+        if (Platform.isZing() || Platform.isJ9() ||
+            (Platform.isGraal() && Platform.isAarch64() && "vm".equals(cstack))) {
```

Comment:
Since this is failing only on ASAN and CI I would do something like
```suggestion
            (isInCI() && isAsan() && Platform.isGraal() && Platform.isAarch64() && "vm".equals(cstack)) {
```